src/Microsoft.AspNetCore.Hosting/Internal/HostingApplication.cs

```diff
@@ -9,11 +9,17 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Http.Features;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Net.Http.Headers;
 
 namespace Microsoft.AspNetCore.Hosting.Internal
 {
     public class HostingApplication : IHttpApplication<HostingApplication.Context>
     {
+#if !NET451
+        private const string ActivityName = "Microsoft.AspNetCore.Hosting.Activity";
```

Comment:
I have submitted a PR https://github.com/dotnet/corefx/pull/17296 to rename System.Net.Http.Activity to Ssytem.Net.Http.HttpReqOut.   I recommend we rename this to Microsoft.AspNetCore.Hosting.HttpReqIn.